packages/gatsby/src/redux/__tests__/index.js

```diff
@@ -221,212 +217,6 @@ describe(`redux db`, () => {
     })
   })
 
-  describe(`Sharding`, () => {
```

Comment:
With LMDB enabled this never ran, so removed it

Reply:
we still do sharding for `pages` and not just `nodes` I think? nodes in redux are in fact empty, but pages are still in redux right now ( pages sharding introduced as part of https://github.com/gatsbyjs/gatsby/pull/28590 )

Reply:
Good catch! So should I remove that early return then or rewrite the tests?

Reply:
probably remove early return and remove the setup / assertions for node sharding, but keep the setup/assertions for pages sharding ( I'm not sure if that would require more rewrites or not )